test/mcmc/Inference.jl

```diff
@@ -44,13 +44,13 @@ using Turing
             end
 
             # Should also be stable with an explicit RNG
-            seed = 5
-            rng = Random.MersenneTwister(seed)
+            local_seed = 5
```

Comment:
This is unrelated to the other changes in this PR, but was discovered as a bug: We were accidentally setting the global seed in a local test.